Aleoris/DRUNK
# Basic drinking architecture in place

How to test:
1. Download Post Processing from the package manager
2. start the game
3. press U to drink a beer (suggest waiting a few seconds between beers to see the decrease effect)
